addons/account_asset/account_asset.py

```diff
@@ -18,7 +18,8 @@ class AccountAssetCategory(models.Model):
     name = fields.Char(required=True, index=True, string="Asset Type")
     account_analytic_id = fields.Many2one('account.analytic.account', string='Analytic Account', domain=[('account_type', '=', 'normal')])
     account_asset_id = fields.Many2one('account.account', string='Asset Account', required=True, domain=[('internal_type','=','other'), ('deprecated', '=', False)])
-    account_income_recognition_id = fields.Many2one('account.account', string='Recognition Income Account', domain=[('internal_type','=','other'), ('deprecated', '=', False)], oldname='account_expense_depreciation_id')
+    account_income_recognition_id = fields.Many2one('account.account', string='Recognition Income Account', domain=[('internal_type','=','other'), ('deprecated', '=', False)], oldname='account_expense_depreciation_id',
+        help="Account used in the periodical entries, to record a part of the asset as expense. But the asset account is used if empty.")
     account_depreciation_id = fields.Many2one('account.account', string='Depreciation Account', required=True, domain=[('internal_type','=','other'), ('deprecated', '=', False)])
```

Comment:
Minor thingy: we could use the correct field labels at the model level as well, to be consistent everywhere, [as in saas-11+](https://github.com/odoo/odoo/blob/10.0/addons/account_asset/models/account_asset.py#L22-L23).
This might also spare us the need to copy those in the view.
Otherwise it looks good to me, thanks!